logger/src/lib.rs

```diff
@@ -95,6 +56,46 @@ impl<D> Logger for Service<D>
 where
     D: Dal + Send + Sync + 'static,
 {
+    async fn store_logs(
+        &self,
+        request: Request<StoreLogsRequest>,
+    ) -> Result<Response<StoreLogsResponse>, Status> {
+        request.verify(Scope::DeploymentPush)?;
+
+        let request = request.into_inner();
+        let logs = request.logs;
+        if !logs.is_empty() {
+            _ = self
+                .logs_tx
+                .broadcast(
+                    logs.into_iter()
+                        .map(|li| {
+                            let timestamp = li.tx_timestamp.clone().unwrap_or_default();
+                            Log {
+                                deployment_id: li.deployment_id,
+                                shuttle_service_name: li.service_name,
+                                tx_timestamp: DateTime::from_utc(
+                                    NaiveDateTime::from_timestamp_opt(
+                                        timestamp.seconds,
+                                        timestamp.nanos.try_into().unwrap_or_default(),
+                                    )
+                                    .unwrap_or_default(),
+                                    Utc,
+                                ),
+                                data: li.data,
+                            }
```

Comment:
Done. I wouldn't implement `From` for this because we would have to do it in `proto` and expose the `shuttle_logger::dal::Log` there. I implemented though `Log::from_stored` method which is doing the same thing.